client/src/pages/SignUp/SignUp.tsx

```diff
@@ -45,7 +45,20 @@ export default function SignUp(): JSX.Element {
           {/* Add a continue button 
                 color can be changed if need be, default is blue*/}
           <Box sx={{ mt: 2, display: 'flex', justifyContent: 'center' }}>
-            <Button variant="contained" size="large" color="success">
+            <Button
```

Comment:
Remember to use the customButton component here.

Reply:
ESLint also wanted me to add keyboard listeners not sure if you got those messages too. But it can be a future PR.